### DISCUSSION TITLE
Code -> download zip should include all linked folders automatically without separate git tools (or have option not to)

### DISCUSSION THREAD

**Discussion by goofyseeker311** [community/community#112095] [Repositories]
### Select Topic Area

General

### Body

by default code -> download zip does not include any linked folders into the zip file, even if its should.
there is no obvious interface feature in web browser to easily include all linked folders as deep download.

you are not expected to install git software package to download all linked directories in the zip?

edit: main point: code -> download as zip does not do what it says it will do. it should download all the things mentioned in the repo. not only the local repo. code download assumed to be full download should include all dependencies linked folders not only this local repo. also if you make a web interface, I assume that interface to be complete not requiring any other tools to get what the web interface is supposed to provide.

**Comment by probablynotnmp** [comment]
The "download zip" function in code doesn't include linked folders, which is expected. There's no obvious web browser feature for deep downloads. While it's not expected to install Git solely for this, using Git or other version control systems can be more efficient for managing and downloading linked directories.

**Comment by goofyseeker311** [comment]
installing even more software just to get the complete package is not efficient at all. more work in vain. a deep zip download option would be better for all. its integrated vs sparse approach, a fat package vs more complex management imo. less tools is more tools. as you can probably read, not a fan of build tools in general, complete integrated environments is better imho.
